src/main/java/gov/usgs/locaux/LocUtil.java

```diff
@@ -152,30 +152,29 @@
   public static final double DEG2KM = 6371d * Math.PI / 180d;
 
   /** An int constant representing the maximum number of iteration stages to attempt. */
-  public static final int STAGELIMIT = 2;
+  /** JMP 1/13/2022 Added a third stage due to large number of events that did not converge */
```

Comment:
Is there any sense of how much this may change how long it takes the locator to run? Or does this only run when it cannot converge after two stages?

Reply:
Not long we're talking 1 more stage of at most 30 iterations. In computer time that doesn't take very long, especially since the iteration loop breaks out after convergence is detected. A 16 phase event that didn't converge before took 1.343 seconds to locate as opposed to 0.703 seconds before.